node/pkg/watchers/sui/watcher.go

```diff
@@ -498,3 +500,18 @@ func (e *Watcher) Run(ctx context.Context) error {
 		return err
 	}
 }
+
+// fixSuiWsURL ensures the websocket scheme is properly specified
+func (e *Watcher) fixSuiWsURL(logger *zap.Logger) error {
+	u, _ := url.Parse(e.suiWS)
+
+	// When the scheme is empty/nil or when the Host is empty but a scheme eis set
```

Comment:
Nit: spelling "eis"

Reply:
Fixed in dd7e1aa772597bbacd1ae7020ea951bde90606d5 thanks!